root_orchestrator/system-manager-python/services/service_management.py

```diff
@@ -44,16 +65,24 @@ def create_services_of_app(username, sla, force=False):
     return {"job_id": str(last_service_id)}, 200
 
 
+# TODO: have logging moved to resource abstractor
```

Comment:
Why was the logging not yet moved?

Reply:
Good question. Well I don't know yet how logging should be tackled. I feel its a discussion that we should have on what should be logged and where. Should every service log everything it's doing?